src/frontend/src/btc/services/btc-send.services.ts

```diff
@@ -35,3 +48,34 @@ export const selectUtxosFee = async ({
 		utxos
 	};
 };
+
+export const sendBtc = async ({ progress, ...rest }: SendBtcParams): Promise<void> => {
+	// TODO: use txid returned by this method to register it as a pending transaction in BE
+	await send({ progress, ...rest });
+
+	progress(ProgressStepsSendBtc.RELOAD);
+
+	await waitAndTriggerWallet();
+};
+
+const send = async ({
+	identity,
+	destination,
+	network,
+	amount,
+	utxosFee,
+	progress
+}: SendBtcParams): Promise<SendBtcResponse> => {
+	const satoshisAmount = convertNumberToSatoshis({ amount });
+	const signerBitcoinNetwork = mapToSignerBitcoinNetwork({ network });
+
+	progress(ProgressStepsSendBtc.SEND);
+
+	return sendBtcApi({
+		identity,
+		network: signerBitcoinNetwork,
+		feeSatoshis: [utxosFee.feeSatoshis],
```

Comment:
nitpicl: instead of `[utxosFee.feeSatoshis]` you can use `toNullable(utxosFee.feeSatoshis)`

Reply:
nice, updated